src/utils/writeSpecs.js

```diff
@@ -1,7 +1,7 @@
 import fs from 'fs';
 
 const writeSpecs = (specPath, specs) => {
-  fs.writeFileSync(specPath, JSON.stringify(specs, null, '  '), 'utf8');
+  fs.writeFileSync(specPath, `${JSON.stringify(specs, null, '  ')}\n`, 'utf8');
```

Comment:
Just a note; you can also replace `'  '` with `2` to say how many spaces each indentation should have:

```js
JSON.stringify(specs, null 2)
```